manage.py

```diff
@@ -14,7 +14,8 @@ def main():
         load_dotenv('./.env')
 
     # When running on Azure App Service you should use the production settings.
-    settings_module = "azureproject.production" if 'RUNNING_IN_PRODUCTION' in os.environ else 'azureproject.development'
+    # settings_module = "azureproject.production" if 'RUNNING_IN_PRODUCTION' in os.environ else 'azureproject.development'
```

Comment:
Remove commented line of code. Oh and comment above!

Reply:
Tx fix on next push